### ISSUE TITLE
Not letting me submit of GitHub

### ISSUE THREAD

**Issue by ryandods** [ryandods/using-markbot#1]
@hellachella

**Comment by hellachella** [comment]
Ok, I'll have a look.

**Comment by ryandods** [comment]
![Screen Shot 2021-01-25 at 9 22 12 AM](https://user-images.githubusercontent.com/77307996/105718080-dbee7f80-5eee-11eb-97e5-6d9013ae4333.png)
![Screen Shot 2021-01-25 at 9 22 17 AM](https://user-images.githubusercontent.com/77307996/105718089-de50d980-5eee-11eb-959c-f32c4357336b.png)

**Comment by hellachella** [comment]
@thomasjbradley

This is the first Markbot assignment, just troubleshooting what is going on. Do you think it's the API key? He is unable to submit.

**Comment by hellachella** [comment]
He is running the right version of Java.

**Comment by thomasjbradley** [comment]
Yeah—it’s either the API key is incorrectly copyed-and-pasted. Or their GitHub username is misspelled—or incorrectly capitalized—inside Markbot.

**Comment by hellachella** [comment]
Is it a matter of just signing out of Markbot and signing back in?

**Comment by thomasjbradley** [comment]
@hellachella Yup!
